src/inter_dc_query_response.erl

```diff
@@ -26,6 +26,8 @@
 %% Description and complete License: see LICENSE file.
 %% -------------------------------------------------------------------
 
+%% @doc Starts a GenServer process and exposes an API to get node entries
+%%      and permissions.
```

Comment:
Short explanation of what node entries and permissions are

Reply:
I couldn't find a lot of docs on this, especially permissions. Could you help me out here?

Reply:
There is no doc on this, as far as I know. `request_permissions` is a special inter dc request for the bounded counter manager.